subprojects/language-native/src/integTest/groovy/org/gradle/language/cpp/CppExecutableIntegrationTest.groovy

```diff
@@ -570,12 +716,12 @@ class CppExecutableIntegrationTest extends AbstractCppInstalledToolChainIntegrat
         succeeds ":app:assemble"
         result.assertTasksExecuted(compileAndLinkTasks([':greeter', ":logger", ':app'], debug), installTaskDebug(':app'), ":app:assemble")
 
-        sharedLibrary("greeter/build/lib/main/debug/greeter").assertExists()
-        sharedLibrary("logger/build/lib/main/debug/logger").assertExists()
-        executable("app/build/exe/main/debug/app").assertExists()
-        installation("app/build/install/main/debug").exec().out == app.expectedOutput
-        sharedLibrary("app/build/install/main/debug/lib/greeter").file.assertExists()
-        sharedLibrary("app/build/install/main/debug/lib/logger").file.assertExists()
+        sharedLibrary("greeter/build/lib/main/debug/shared/greeter").assertExists()
+        sharedLibrary("logger/build/lib/main/debug/shared/logger").assertExists()
+        executable("app/build/exe/main/debug/shared/app").assertExists()
+        installation("app/build/install/main/debug/shared").exec().out == app.expectedOutput
+        file("app/build/install/main/debug/shared/lib").assertHasDescendants([
```

Comment:
should add something to the `NativeInstallationFixture` to do this assertion, rather than duplicating knowledge of the layout in all of the tests.

Reply:
Done.